arex-instrumentation-api/src/main/java/io/arex/inst/runtime/util/ReplayUtil.java

```diff
@@ -0,0 +1,127 @@
+package io.arex.inst.runtime.util;
+
+import io.arex.agent.bootstrap.model.ArexMocker;
+import io.arex.agent.bootstrap.model.MockCategoryType;
+import io.arex.agent.bootstrap.model.Mocker;
+import io.arex.agent.bootstrap.util.CollectionUtil;
+import io.arex.inst.runtime.context.ContextManager;
+import io.arex.inst.runtime.log.LogManager;
+import io.arex.inst.runtime.model.ArexConstants;
+import io.arex.inst.runtime.model.MergeDTO;
+import io.arex.inst.runtime.model.QueryAllMockerDTO;
+import io.arex.inst.runtime.serializer.Serializer;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Map;
+import java.util.function.Predicate;
+
+/**
+ * Replay all mockers under case
+ * (called once after arexContext init)
+ */
+public class ReplayUtil {
+
+    /**
+     * init replay all mocker under case and cached replay result
+     */
+    public static void queryMockers() {
+        if (!ContextManager.needReplay()) {
+            return;
+        }
+        try {
+            QueryAllMockerDTO requestMocker = new QueryAllMockerDTO();
+            requestMocker.setRecordId(ContextManager.currentContext().getCaseId());
+            requestMocker.setCategoryTypes(new String[]{MockCategoryType.DYNAMIC_CLASS.getName(), MockCategoryType.REDIS.getName()});
```

Comment:
This can be declared as `static final`. 
`new String[]{MockCategoryType.DYNAMIC_CLASS.getName(), MockCategoryType.REDIS.getName()}`

Reply:
get